packages/webapp/src/locales/i18n.js

```diff
@@ -12,7 +12,15 @@ i18n
     defaultNS: 'translation',
     nsSeparator: ':',
     fallbackLng: 'en',
-    locales: ['en', 'pt', 'es', 'fr', 'de', 'hi', 'pa', 'ml'],
+    supportedLngs: ['en', 'pt', 'es', 'fr', 'de'],
```

Comment:
The i18n allowlist, see [i18n configuration options](https://www.i18next.com/overview/configuration-options)